backend/src/test/java/com/benine/backend/http/EditPresetHandlerTest.java

```diff
@@ -0,0 +1,108 @@
+package com.benine.backend.http;
+
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import java.io.BufferedInputStream;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.List;
+
+import org.eclipse.jetty.util.MultiMap;
+import org.junit.Before;
+import org.junit.Test;
+
+import com.benine.backend.Preset;
+import com.benine.backend.PresetController;
+import com.benine.backend.ServerController;
+import com.benine.backend.camera.CameraConnectionException;
+import com.benine.backend.camera.Position;
+import com.benine.backend.camera.ipcameracontrol.IPCamera;
+import com.benine.backend.video.MJPEGStreamReader;
+import com.benine.backend.video.Stream;
+import com.benine.backend.video.StreamNotAvailableException;
+
+public class EditPresetHandlerTest { //extends RequestHandlerTest {
```

Comment:
Then I think you should solve the issues with Eclipse before pushing or exclude the test file from your commit.